### PR TITLE
Awesome Books: A complete website with navigation.

### PR DESCRIPTION
**Pull request by Fene-87**
Create a website that allows a user to add or remove books.
-Add a navigation bar that changes the section to be displayed on the page.
-Add a book list section, a form section to add a new book, and a contact information section.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -102,3 +102,62 @@ document.querySelector('.displayed-books').addEventListener('click', (e) => {
...
+const firstSection = document.querySelector('.first-section');
+const secondSection = document.querySelector('.second-section');
+const thirdSection = document.querySelector('.third-section');
+
+navLinks.forEach((link, index) => {
+  if(index === 0) {
+    link.addEventListener('click', () => {
+      link.classList.add('link-color');
+      link[1].classList.remove('link-color');
+      link[2].classList.remove('link-color');
```
Comment (julie-ify) [inline app.js]: - [ ] When I navigate to the "Add New" and "Contact" sections I cannot navigate back to the "List"

  image

  You can fix this by changing `link` to `navLinks`

  ```
  navLinks[1].classList.remove('link-color');
  navLinks[2].classList.remove('link-color');
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Fene-87,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (Fene-87) [reply app.js]: Thanks. That is now fixed.

**Review by EricMbouwe — APPROVED** [review]
Hi @Fene-87, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**